src_new/adapters/prebid.js

```diff
@@ -515,6 +515,8 @@ function fetchBids(activeSlots, impressionID){
 
 	var adUnits = {};// create ad-units for prebid
 	var randomNumberBelow100 = adapterManager.getRandomNumberBelow100();
+	var isRubiconPresent = false;
```

Comment:
add a list of such adapters, we should not be adding a variable per adapter

Reply:
Done